diff -urN libogg-1.0rc1.org/configure.in libogg-1.0rc1/configure.in
--- libogg-1.0rc1.org/configure.in	Wed Jun 27 12:09:19 2001
+++ libogg-1.0rc1/configure.in	Wed Jun 27 12:10:17 2001
@@ -41,8 +41,8 @@
         case $host in 
         *-*-linux*)
                 DEBUG="-g -Wall -fsigned-char"
-                CFLAGS="-O20 -ffast-math -fsigned-char"
-                PROFILE="-Wall -W -pg -g -O20 -ffast-math -fsigned-char"
+                CFLAGS="$CFLAGS -ffast-math -fsigned-char"
+                PROFILE="$CFLAGS -Wall -W -pg -g -ffast-math -fsigned-char"
 		;;
         sparc-sun-*)
                 DEBUG="-g -Wall -fsigned-char -mv8"
@@ -56,8 +56,8 @@
 		;;
         *)
                 DEBUG="-g -Wall -fsigned-char"
-                CFLAGS="-O20 -fsigned-char"
-                PROFILE="-O20 -g -pg -fsigned-char" 
+                CFLAGS="$CFLAGS -fsigned-char"
+                PROFILE="$CFLAGS -g -pg -fsigned-char" 
 		;;
         esac
 fi
